Functions-Challenge.playground/Contents.swift

```diff
@@ -29,14 +29,21 @@
 */
 
 
+func calculateMortysGrade() -> Double {
+    let homework = [100, 100, 100, 85, 65, 100, 100, 100, 0, 105, 105]
```

Comment:
While this is the cleanest solution -- students do not know `Array` or `reduce` yet.